vllm/entrypoints/cli/serve.py

```diff
@@ -148,7 +144,7 @@ def signal_handler(signum, frame):
         start_index=args.data_parallel_start_rank,
```

Comment:
Does `data_parallel_start_rank` need to be set at all with this approach?

Reply:
No it doesn't, this line though is in the headless path (in `run_headless` function), used for remote nodes in the internal LB mode.